0028-web-monetization/0028-web-monetization.md

```diff
@@ -1,79 +1,296 @@
 ---
 title: Web Monetization
-draft: 2
+draft: 3
 ---
 
 # Web Monetization
 
-Web Monetization is a proposed browser API that uses ILP micropayments to monetize a site. It can be polyfilled by extensions, or can be implemented directly into an ILP-enabled browser.
+Web Monetization is a proposed browser API that uses ILP micropayments to monetize a site. It can be provided through a polyfill or an extension, but the goal is to eventually implement it directly into the browser.
 
 ## Overview
 
 ### Terminology
 
 - The **webmaster** is the party who is running a site.
 - The **user** is the party who is accessing the site.
+- The **provider** is the party providing Interledger access to the user.
 
 ### Design Goals
 
+- Should work on mobile and desktop without requiring an extension or special browser.
 - Should be extremely simple for webmasters to use in their site.
 - Backend infrastructure should be optional; should be usable on a static site.
 - Should not require any interaction with the user.
 - Should give user's agent a choice about how much to spend, and which sites to support.
 - Should give advanced webmasters a way to associate payments with their users, in order to unlock premium experiences.
 - Should pay continuously as the user consumes content.
+- Should be compatible with existing application and transport protocols on Interledger.
 
 ### Relation to Other Protocols
 
 The reason this is not using the W3C Web Payments API is that Web Monetization is intended for continuous payments rather than discrete payments. It is also not designed to have any user interaction. The idea is to provide a direct alternative to advertisements, rather than an alternative to existing checkout methods.
 
 With advertisements, the browser decides whether to display the ads and the user decides whether to engage with the ads. With Web Monetization, the browser decides whether to pay the site and, if so, how much to pay.
 
-Web Monetization makes use of [Payment Pointers](../0026-payment-pointers/0026-payment-pointers.md) in order to associate a site with an ILP destination, and [SPSP](../0009-simple-payment-setup-protocol/0009-simple-payment-setup-protocol.md) in order to set up ILP payments.
-
-The browser or browser extension which provides Web Monetization will likely use the [Ledger Plugin Interface](../0004-ledger-plugin-interface/0004-ledger-plugin-interface) in order to trigger payment when a site requests it.
+Web Monetization makes use of [ILP/STREAM](../0029-stream/0029-stream.md) provide a high-level way to send money and data, while still providing tremendous flexibility.
```

Comment:
Missing a "to" after the link

Reply:
It would be worth describing what type of flexibility this provides (e.g. opening connections to multiple destinations)

On that note, can a site open an unlimited number of connections?